### DISCUSSION TITLE
LDM Notes for June 6th, 2022

### DISCUSSION THREAD

**Discussion by MadsTorgersen** [dotnet/csharplang#6219] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-06-06.md

## Agenda

1. Open issues for `ref` fields
2. Open issues for static virtual members
3. Concatenation of Utf8 literals

**Comment by huoyaoyuan** [comment]
> No diagnostics - it's not worth it, wouldn't save every case anyway, and this is an expert feature.

I'd expect least support like NRT fields - experts do need assist too.
